EmotiBitOscilloscope/src/ofApp.cpp

```diff
@@ -52,6 +53,121 @@ void ofApp::update() {
 	updateMenuButtons();
 }
 
+void ofApp::setTypeTagPlotAttributes()
+{
+	// ToDo: Add attributes for all streams for refactor
+	{
+		// Add plot attributes for THERMOPILE data
+		typeTagPlotAttr attr;
+		attr.tapeTagName = "THERM";
+		attr.typeTagColor = ofColor(239, 97, 82);
+		// ToDo: someday, we can even consider a standard layout considering even plot number p -> {w,s,p}
+		std::vector<int> sIdx = { 1, 3 };  // {window, scope}
+		attr.scopeIdx = sIdx;
+		typeTagPlotAttributes.emplace(EmotiBitPacket::TypeTag::THERMOPILE, attr);
+	}
+
+	{
+		// Plot Attributes for TEMP1 data
+		typeTagPlotAttr attr;
+		attr.tapeTagName = "TEMP1";
+		attr.typeTagColor = ofColor(234, 174, 68);
+		std::vector<int> sIdx = { 1, 3 };  // {window, scope}
+		attr.scopeIdx = sIdx;
+		typeTagPlotAttributes.emplace(EmotiBitPacket::TypeTag::TEMPERATURE_1, attr);
+	}
+}
+
+void ofApp::resetScopePlot(int w, int s)
+{
+	scopeWins.at(w).scopes.at(s).clearData();
```

Comment:
There should not be a mismatch, because, on every `processData` cycle, the latest packet data gets added to the buffers.
In my opinion, we also don't need to clear all other scopes when updating just one scope.